task/upload.go

```diff
@@ -407,6 +423,15 @@ func outputLocations(outURL string, relativePath string) ([]OutputName, []client
 				},
 			},
 		}
+	names, locations =
```

Comment:
Actually, this should be under `assetOutputLocations`, not `outputLocations`, since we don't want this output in the case of the Transcode API.

Reply:
Good point! Moved it :)